masterlist.yaml

```
@@ -10052,11 +10145,14 @@ plugins:
 
   - name: 'SkyrimsUniqueTreasures.esp'
     url: [ 'https://www.nexusmods.com/skyrimspecialedition/mods/5261/' ]
+    msg:
+      - <<: *patch3rdParty_LotD
+        condition: 'active("LegacyoftheDragonborn.esm") and not active("DBM_SUT_Patch.esp")'
     clean:
-      # version: 4.4
+     # v4.4
```

I'd just change it back to remove noise from your pull request.
```suggestion
      # version: 4.4
```